### PR TITLE
Publish additional app configs for {major} and {major}.{minor} releases

### PR DESCRIPTION
**Pull request by j3parker**
Thoughts?

@dlockhart @dbatiste 

Gulp isn't really my jam, I think this was a bit ham-fisted...

Example output:

https://s.brightspace.com/apps/saml-configuration/appconfig.v1.json
https://s.brightspace.com/apps/saml-configuration/appconfig.v1.0.json
https://s.brightspace.com/apps/saml-configuration/appconfig.v1.1.json

Example publish:

https://travis-ci.com/Brightspace/saml-configuration-ui/builds/27185379#L1362-L1403

### CODE REVIEW

**Comment by j3parker** [comment]
Hm, the travis output is different than on my dev machine. I'll investigate.

**Comment by dlockhart** [comment]
LGTM. Only thing that might be missing is if you wanted to add any tests to cover the new behaviour.

**Comment by j3parker** [comment]
Oh yeah, good idea :)

File: package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "gulp": "^3.8.10",
     "istanbul": "^0.4.2",
     "mocha": "^2.0.1",
+    "multipipe": "^0.3.0",
```
Comment (Surekh) [inline package.json]: Should this not be a dev-dependency?

```
Error: Cannot find module 'multipipe'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/travis/build/Brightspace/sequence-viewer/node_modules/frau-publisher/src/publisher.js:10:9)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
```

Comment (dlockhart) [inline package.json]: @j3parker I think @Surekh is correct, `multipipe` should be a normal dependency.

**Comment by j3parker** [comment]
Before merging I'm going to add some tests that validate the stuff going through gulp because that stuff has gotten complicated.

**Comment by j3parker** [comment]
The official solution is in [cdn-manager](https://github.com/Brightspace/cdn-manager). It's not quite as `--fast` as this one but has better checks and balances/room for evolution. I'll hunt down which apps were using my fork and switch them back to the official publisher.

**Pull request closed without merging**